src/main/java/svc/managers/SMSAlertManager.java

```diff
@@ -0,0 +1,94 @@
+package svc.managers;
+
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import svc.data.textMessages.SMSAlertDAO;
+import svc.data.textMessages.SMSAlertMessageCreator;
+import svc.dto.CitationSearchCriteria;
+import svc.models.Citation;
+import svc.models.Court;
+import svc.models.SMSAlert;
+import svc.models.SMSAlertNotification;
+import svc.util.DatabaseUtilities;
+
+@Component
+public class SMSAlertManager {
+	@Inject
+	private SMSAlertDAO smsAlertsDAO;
+	
+	@Inject
+	private CitationManager citationManager;
+	
+	@Inject
+	private CourtManager courtManager;
+	
+	@Value("${spring.clientURL}")
```

Comment:
Yes, there was still a old reference in the yaml.  That's why it still worked.  Changed both.